src/SourceWatcherFuncApp/.gitignore

```diff
@@ -0,0 +1,265 @@
+## Ignore Visual Studio temporary files, build results, and
+## files generated by popular Visual Studio add-ons.
+
+# Azure Functions localsettings file
+local.settings.json
+configuration.json
```

Comment:
Maybe you can just add the the function specific things that need to be ignored in the .gitignore file in the root directory.